tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -93,8 +93,17 @@ driver = "file"
 # keep-after-success = false
 
 [tikv-importer]
+<<<<<<< HEAD
```

Comment:
@glorv Can you take a look at these conflicts? I'm not sure which is correct for v4.0 ~ v5.3

Reply:
done